bitmark-references/bitmark-sdk/migration.md

```diff
@@ -0,0 +1,32 @@
+# Account migration
+
+Bitmark allows users migrate their own account from previous twenty four recovery phrase words to new twelve one.
+
+The method requires a valid 24 recovery phrase words and return new `Account` object with 12 recovery phrase words as well as a new list bitmark id.
+
+```javascript
+// Not supported at the moment. If you have demand for this function, please feel free to contact us.
+```
+
+```swift
+let phrase = ["abuse", "tooth", "riot", "whale", "dance", "dawn", "armor", "patch", "tube", "sugar", "edit", "clean",
+                "guilt", "person", "lake", "height", "tilt", "wall", "prosper", "episode", "produce", "spy", "artist", "account"]
+let (account, bitmarkids) = try Migration.migrate(recoverPhrase: phrase, language: .english)
+```
+
+```java
+final String[] phrase = new String[] {"abuse", "tooth", "riot", "whale", "dance", "dawn", "armor", "patch", "tube", "sugar", "edit", "clean","guilt", "person", "lake", "height", "tilt", "wall", "prosper", "episode", "produce", "spy", "artist", "account"};
+
+Migration.migrate(phrase, new Callback1<Pair<Account, List<String>>>() {
+                @Override
+                public void onSuccess(Pair<Account, List<String>> data){
+                    final Account account = data.first();
+                    final List<String> bitmarkIds = data.second();
+                }
+
+                @Override
+                public void onError(Throwable throwable) {
+                	// Throwable goes here
+                }
+            });
+```
```

Comment:
@nodestory, @jollyjoker992, @anhnguyenbitmark: I think this function has been removed and replaced by the `rekey` function.

Reply:
I will update it after i finish the implementation.

Reply:
Updated for Java SDK